tests/rc.rs

```diff
@@ -22,7 +24,63 @@ fn test_basic() {
 #[cfg(feature = "futures03")]
 #[test]
 fn test_stream() {
+    use futures::executor::block_on_stream;
+
     let gen = Gen::new(odd_numbers_less_than_ten);
     let xs: Vec<_> = block_on_stream(gen).collect();
     assert_eq!(xs, [1, 3, 5, 7, 9]);
 }
+
+#[cfg(feature = "proc_macro")]
+#[test]
+fn rc_proc_macro_fn() {
+    #[genawaiter::rc::rc_yield_fn(u8)]
+    async fn odds() {
+        for n in (1_u8..).step_by(2).take_while(|&n| n < 10) {
+            genawaiter::yield_!(n);
```

Comment:
Yea I agree I did change it all then realized the reason I did it was the `strict` lint fails because when the `proc_macro` expands it warns that `yield_!` is not used.